src/com/palmergames/bukkit/towny/permissions/PermissionNodes.java

```diff
@@ -79,7 +81,13 @@
 			TOWNY_COMMAND_TOWN_SET_NAME("towny.command.town.set.name"),
 			TOWNY_COMMAND_TOWN_SET_TAG("towny.command.town.set.tag"),
 			TOWNY_COMMAND_TOWN_SET_JAIL("towny.command.town.set.jail"),
-		
+			TOWNY_COMMAND_TOWN_SET_WARP("towny.command.town.set.warp"), //TODO: Finish Implementation

```

Comment:
All of these TODOs, did they get done? Do the commands still need to have permission checks done on them?